datus-hive/datus_hive/connector.py

```
@@ -332,8 +338,7 @@ def full_name(
     def do_switch_context(self, catalog_name: str = "", database_name: str = "", schema_name: str = ""):
```

Ruff (0.15.2)

[warning] Unused method argument: `catalog_name`

(ARG002)